### PR TITLE
Feature 206: UI Sidebar & Header anpassen

### LINKED ISSUE #206: UI Sidebar & Header anpassen
**Beschreibung**

Passe Sidebar und Header entsprechend des Figma Entwurfs an.

**Acceptance Criteria**

- [x] Icons stimmen überein mit Text on hover
- [x] Language selection wird dropdown
- [x] #119 User oben rechts
- [x] Direkter Butten in sidebar für User erstellen
- [x] Zahnrad links unten ist Admin View inklusive Elastic Sync Button und User + Workgroup Management
- [x] Suchfeld Aussehen angepasst
- [x] Suchfilter muss korrekt über Suchresultaten liegen - Z-Index erhöhen damit Schatten korrekt angezeigt werden

### PR DESCRIPTION
**Pull request by daniel-va**
Resolves #206.

### CODE REVIEW

**Review by TIL-EBP — COMMENTED** [review]
General Comments before looking at the code: 
- The active tab in the sidebar is not highlighted (according to Figma, it should be green)
-  Nitpick: When hovering over the language selection dropdown, the color is off, it should be light-green (same color as when hovering over the logou button).
- Regarding this point "Suchfilter muss korrekt über Suchresultaten liegen - Z-Index erhöhen damit Schatten korrekt angezeigt werden", I dont see a change to dev, but I am also not quite sure what should have changed, it looks fine to me. 
- Super nitpick: The stars in the table of the reuslts to add as favorite have changed thickness, was this intentional?

File: libs/asset-editor/src/lib/components/asset-editor-launch/asset-editor-launch.component.html [hunk 1]
```diff
@@ -3,12 +3,14 @@
   <asset-sg-drawer>
     <asset-sg-drawer-panel widthPercentageOfViewPanel="0.3" [display]="true">
       <div class="flex flex-column pl-2 pt-4 drawer-panel">
-        <h3 translate>edit.adminInstructionsEditHeading</h3>
-        <p translate class="m-0">edit.adminInstructionsEdit</p>
-        <h3 translate class="mt-8">edit.adminInstructionsCreateHeading</h3>
-        <a asset-sg-primary routerLink="new" class="self-start" translate>edit.adminInstructionsCreate</a>
+        <ng-container *rxLet="currentLang$; let currentLang">
+          <h3 translate>edit.userManagementHeading</h3>
+          <a asset-sg-primary [routerLink]="[currentLang, 'admin', 'users']" class="self-start" translate>
```
Comment (TIL-EBP) [inline libs/asset-editor/src/lib/components/asset-editor-launch/asset-editor-launch.component.html:8]: this doesnt seem to work, i t redirects me to the main page

File: apps/client-asset-sg/src/app/components/menu-bar-item/menu-bar-item.component.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  @Input({ required: true })
+  icon!: string;
+
+  @Input()
+  link: string[] | null = null;
+
+  @Input({ transform: coerceBooleanProperty })
+  disabled = false;
+
+  @HostBinding('aria-disabled')
```
Comment (TIL-EBP) [inline apps/client-asset-sg/src/app/components/menu-bar-item/menu-bar-item.component.ts:19]: Just a question for my understanding: Why use hostBInding and bind inside the tempalte?

File: apps/client-asset-sg/src/app/components/app-bar/app-bar.component.html [hunk 3]
```diff
@@ -1,16 +1,49 @@
-<a routerLink="/">
-  <img class="logo" src="../../../assets/swissgeol_assets.svg" alt="{{ 'logoSwissGeol' | translate }}" />
+<a routerLink="/" class="logo">
+  <img src="../../../assets/swissgeol_assets.svg" alt="{{ 'logoSwissGeol' | translate }}" width="130" height="36" />
```
[the remark this answers is not in the document]
Comment (daniel-va) [reply apps/client-asset-sg/src/app/components/app-bar/app-bar.component.html:2]: This helps the browser reserve the space for the image before it has loaded in. This makes the layout not shift around on image load.

File: apps/client-asset-sg/src/app/components/app-bar/app-bar.component.scss [hunk 4]
```diff
@@ -1,52 +1,143 @@
...
+  color: #828e9a;
+}
+
+button.language {
+  text-transform: uppercase;
+  width: 69px;
+  height: 36px;
+  padding: 8px 12px;
+  border-radius: 4px;
+  color: #337083;
```
[the remark this answers is not in the document]
Comment (daniel-va) [reply apps/client-asset-sg/src/app/components/app-bar/app-bar.component.scss:31]: We decided that we want to make a full refactoring of all style variables sometime in the future (this year, if possible). For now, I hard coded the new styles, with full intention to extract them later.

File: apps/client-asset-sg/src/app/components/menu-bar-item/menu-bar-item.component.ts (hunk 2 above)
Comment (daniel-va) [reply apps/client-asset-sg/src/app/components/menu-bar-item/menu-bar-item.component.ts:19]: Host bindings add attributes to the component's host element (in this case, a `li`). That element is not defined in the component's template, but rather specified when the component is used (e.g. `<li asset-sg-menu-bar-item>...</li>`. There is no way to modify that element from within the component's template itself.

**Review by TIL-EBP — APPROVED** [review]

File: apps/client-asset-sg/src/app/components/menu-bar-item/menu-bar-item.component.ts (hunk 2 above)
Comment (TIL-EBP) [reply apps/client-asset-sg/src/app/components/menu-bar-item/menu-bar-item.component.ts:19]: I see, thanks for clarifying

**Pull request merged**